sel4-ftrace-merge/include/ftrace.h

```diff
@@ -0,0 +1,213 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+/*
+ * Copyright 2023, Technology Innovation Institute
+ *
+ */
+#ifndef FTRACE_H
+#define FTRACE_H
+#include <stdint.h>
+#include <stddef.h>
+
+#define read_sized_data(T, d, f) do			\
```

Comment:
That opening brace belongs on the same line as `do`. The only exception (that comes to my mind right now) is function definition where the brace is on its own line.

Reply:
Sure, moved on the same line.